### ISSUE TITLE
2-3 second lag while performing actions since 1.7

### ISSUE THREAD

**Issue by mikmik111** [Anveio/mturk-engine#36]
# Issue summary

Since 1.7 there is a noticeable lag when I perform some actions like clicking the tabs at the top (i.e. Queue, Watchers, etc.), and adding some hits to the queue. 

## Expected behavior

Clicking through the different tabs at the top like Queue, Watchers, Search should be instantaneous and some other buttons should respond instantly.

## Actual behavior

I'm getting a noticeable 2-3 second lag between each button or tab I am pressing, which was not the case before.

## Steps to Reproduce the Problem

1. Click on the Queue tab.
2. Notice that it has a noticeable 2-3 second delay
3. Go back to the Search tab and notice it has a delay moving back to the tab as well. 

## Specifications

* Mturk Engine version: 1.7 
* Browser: Opera 52.0.2871.99

**Comment by Anveio** [comment]
Thanks for posting this issue @mikmik111, I just released an update that hopefully fixes this performance issue. Let me know if you're still experiencing this bug.

**Comment by mikmik111** [comment]
Hi! Thanks for that quick update. I was able to update to the latest one, 1.7.1, and initially there were no improvements to the performance. I went ahead and did a backup, had a clear data with Opera and restarting seems to have solved the issue. I decided to clear my blocklist, I had more than 3500 in there, and that seems to have solved the performance issue for me. Prior to 1.7, having a large blocklist seem to not affect the performance. I hope this helps! Thank you!
